ls1intum/Artemis
# Next assessment text exercise

### Checklist

### Motivation and Context
When we corrected homework, we noticed, after a submission is done there is a “Asses Next Submission” button for the model exercises but not for the text exercises. Because it takes a lot of unnecessary time to return to the Exercise Dashboard, scroll down and click on Asses next Submission, we decided to implement the “Asses Next Submission” button also for the text exercises to have the same functionality there.


### Description

It will be possible to start assessing a the next submission without returning to the exercise dashboard. The new button appears automatically after the assessment is submitted. A click on that button leads directly to a not assessed submission, following the same logic as the functionality concerning the modelling exercises, where such a button already exists. 
If one reached the last submission and therefore, a next student's submission is not available, a click on the new button leads to an altert reporting that there are no submissions left. 

### Steps for Testing

1. Log in to ArTEMiS
2. Submit some example solutions
3. Navigate to Course Administration
4. Select the text exercise
5. Click on Start new assessment
6. Grade the submission
7. Click on submit assessment
8. The "Asses Next Submission" appears
9. Click on the new button "Asses Next Submission"
10. New unassessed submission loads
11. Perform those steps for all further submission
12. If one reached the last submission to grade, and one clicks the new button, an alert occurs that there are no more unassessed submissions left  

### Screenshots



![videoFinal](https://user-images.githubusercontent.com/41108487/60023124-793f5d00-9695-11e9-9610-729a274a4c1f.gif)
